packages/meteor-gazelle-createdby/lib/model.js

```diff
@@ -0,0 +1,32 @@
+Astro.createBehavior({
+  name: 'createdBy',
+  events: {
+    addbehavior: function (behaviorData) {
+      var self = this;
+
+      self.addFields({
+        createdBy: {
+          type: 'string',
+          default: null
+        },
+        updatedBy: {
+          type: 'string',
+          default: []
+        }
+      });
+
+      self.addEvents({
+        beforeInsert: function () {
+          self.createdBy = Meteor.userId();
+        },
+        beforeUpdate: function () {
+          self.updatedBy.push(Meteor.userId());
```

Comment:
Should carry a timestamp as well. Also this may have overlap with revision history.